deploy/cephfs/kubernetes/csi-nodeplugin-rbac.yaml

```diff
@@ -3,3 +3,46 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: cephfs-csi-nodeplugin
+  namespace: default
```

Comment:
use `namespace: {{ .Release.Namespace }}` here

Reply:
The RBD analogue doesn't have that construct. It is for use in the helm templates, not?